File: availability/healthcheck.py
import http.client

connection = http.client.HTTPSConnection("www.fitcalcs.life")
connection.request("GET", "/")
response = connection.getresponse()
statusCode = response.status
data = response.read().decode("utf-8")
print(f' * * * Status Code: {statusCode} * * *')
print(' * * * Data: --v-- * * * ')
print(data)

File: availability/healthcheck.rs
use reqwest::Client;

async fn fetch_data() -> Result<String, Box<dyn std::error::Error>> {
    let client = Client::new();
    let response = client.get("https://www.fitcalcs.life").send().await?;

    if response.status().is_success() {
        let data = response.text().await?;
        Ok(data)
    } else {
        Err(format!("* * * Error fetching data: {}", response.status()).into())
    }
}